.github/workflows/testingfarm.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Testing farm tests
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  testingfarm:
+    name: "Run in testing farm"
+    runs-on: ubuntu-latest
+    if: |
+      github.event.issue.pull_request
+      && contains(github.event.comment.body, '[test]')
```

Comment:
It can be executed only in case the comment contains [test]. It can be removed ;)

Reply:
I think it is good to have it for the start. Maybe I'd use '[test-tmt]' to be more specific.